<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">ImageShack Uploader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images_rc.qrc">
    <normaloff>:/images/images/windowsicon.ico</normaloff>:/images/images/windowsicon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="LoginWidget" name="loginWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="title">
       <string>Image and video uploading</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="midLineWidth">
          <number>2</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="MediaListWidget" name="mediaList" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="ImageInfoWidget" name="imageInfo" native="true"/>
           </item>
           <item>
            <widget class="TagWidget" name="tagWidget">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Privacy &amp;&amp; tags</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="ProgressWidget" name="progressWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>60</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>738</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptione">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSupport"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptione"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionSupport">
   <property name="text">
    <string>Support</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Check for updates</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ProgressWidget</class>
   <extends>QWidget</extends>
   <header>progresswidget.h</header>
   <container>1</container>
   <slots>
    <signal>upload()</signal>
    <signal>cancel()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>MediaListWidget</class>
   <extends>QWidget</extends>
   <header>medialistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LoginWidget</class>
   <extends>QWidget</extends>
   <header>loginwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TagWidget</class>
   <extends>QGroupBox</extends>
   <header>tagwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageInfoWidget</class>
   <extends>QWidget</extends>
   <header>imageinfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images_rc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>progressWidget</sender>
   <signal>upload()</signal>
   <receiver>MainWindow</receiver>
   <slot>upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>513</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSupport</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>openSupport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>checkUpdates()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOptions</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showOptions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>289</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>imageSelected(QModelIndex)</slot>
  <slot>upload()</slot>
  <slot>thumbnailClicked()</slot>
  <slot>showAbout()</slot>
  <slot>openSupport()</slot>
  <slot>checkUpdates()</slot>
  <slot>showOptions()</slot>
 </slots>
</ui>
